lib/linter.sh

```diff
@@ -3,7 +3,7 @@
 
 ################################################################################
 ################################################################################
-########### Super-Linter (Lint all the code) @AdmiralAwkbar ####################
+########### Super-Linter (Lint all the code) @Admiralawkbar ####################
```

Comment:
inconsistent with other changes
```suggestion
########### Super-Linter (Lint all the code) @admiralawkbar ####################
```